webapp/src/components/App/Singlestudy/explore/Configuration/General/dialogs/ScenarioPlaylistDialog/index.tsx

```diff
@@ -98,7 +100,7 @@ function ScenarioPlaylistDialog({ study, open, onClose }: Props) {
       (acc, [_, { status, weight }]) => {
         if (status) {
           acc.selected += 1;
-          acc.sumWeights += weight;
+          acc.sumWeights = new Decimal(acc.sumWeights).plus(weight).toNumber();
```

Comment:
Using a lib for this only seems not ideal, do you have any reason to not do this in plain JS ? 
I dont know the issue you having, but have you tried something like this `acc.sumWeights = Number.parseFloat((acc.sumWeights + weight).toFixed(2));`

or if you dealing with floating point errors you can also consider Number.EPSILON 
```ts
function roundToPrecision(num, decimals = 2) {
  const multiplier = 10 ** decimals;
  return Math.round((num + Number.EPSILON) * multiplier) / multiplier;
}


acc.sumWeights = roundToPrecision(acc.sumWeights + weight);
```

Reply:
I must not round the result (I already asked). Vanilla JS gives wrong values with float values in some case:
e.g. `0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1+0.1` gives `0.999999....`.
The lib allow to fix this issue.